src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,7 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.toLowerCase().replaceAll("[^a-zA-Z0-9]", "");
```

Comment:
if you have used toLowerCase() do you need to replace A-Z then?

Reply:
i fixed it